frontend/.env

```
@@ -1,8 +1 @@
-VITE_CONTRACT_ADDRESS=0x5FbDB2315678afecb367f032d93F642f64180aa3
-VITE_NETWORK_ID=1
-VITE_INFURA_ID=your_infura_project_id
-VITE_IPFS_GATEWAY=https://ipfs.io/ipfs/
-
-
-
-SECERET_KEY_WALLET = start mean stay three meat degree shrimp slow reduce rain guide disease
\ No newline at end of file
+VITE_CONTRACT_ADDRESS=0x5FbDB2315678afecb367f032d93F642f64180aa3
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)